<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_serv_man_config</class>
 <widget class="QDialog" name="Dlg_serv_man_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>厂家配置</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>趋势回顾长度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ok">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_bianmaqi">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>45</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>15</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>语言</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>串口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>15</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>体温通道</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>225</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>波形回顾长度</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_boxingchangdu">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="m_cmbb_pwr_board">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>电源板</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_shuchu">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_jiluyi">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>255</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>记录仪 </string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_spo2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>295</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_para_switch">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>339</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>参数开关</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_cmbb_tfp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>85</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TFP尺寸</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>85</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IBP开关</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>编码器</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_qushihuigu">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>155</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>语音报警</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>225</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>输出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>295</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Protocol</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_chumoping">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>185</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_yuyinbaojing">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_xinxi">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>325</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>信息</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_usb">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_tiwentongdao">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>295</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SPO2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_protocol">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>触摸屏</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_tfp_val">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>4</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_pwr_board_val">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>115</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>5</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_lang">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_yonghuxinxi">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>255</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>4</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_chuankou">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>185</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>用户信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_cancel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>155</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>USB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>可选模块</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_kexuanmokuai">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>115</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_oem">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>45</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_ibp">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>OEM厂家</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
